PromptBox/data/tables/proBox-sct.tbm

```diff
@@ -463,6 +463,8 @@ end
 
 function PromptBox:Draw()
 
+	gr.CurrentOpacityType = GR_ALPHABLEND_NONE
```

Comment:
for safety's sake I'd ask that you store what the opacity type was before this and reset it back to that after drawing. You never know when some weird thing will make assumptions about this shared state...

Reply:
Fair point.  Updated.